controller/proxy-injector/webhook.go

```diff
@@ -152,10 +152,13 @@ func (w *Webhook) inject(request *admissionv1beta1.AdmissionRequest) (*admission
 	patch.addVolume(caBundle)
 	patch.addVolume(tlsSecrets)
 
-	patch.addPodLabel(map[string]string{
-		k8sPkg.ControllerNSLabel:    w.controllerNamespace,
-		k8sPkg.ProxyDeploymentLabel: deployment.ObjectMeta.Name,
-	})
+	labels := map[string]string{}
```

Comment:
Would it be simpler to just mutate the `deployment.Spec.Template.Labels` map rather than allocating a new one?

Reply:
I just mean mutating the `deployment.Spec.Template.Labels` map in memory and then calling `patch.addPodLabels(deployment.Spec.Template.Labels)` rather than allocating a new map.  I figured this might be simpler and more direct.  
But what you have here is fine too, I was just curious.

Reply:
Ah.. gotcha. Yes, you are right 👍 It's ok to mutate `deployment` here. If this PR isn't merged after I get off work today, I can change it 😄